### ISSUE TITLE
Free tier details

### ISSUE THREAD

**Issue by RyanTG** [Engagespot/heroku-free-alternatives#21]
Not sure if this should be in the Description field, or have new fields for this. But people seem interested in whether a credit card is required.

Also, might be worth including known limitations to the free tier? Such as Render deleting postgresql databases after x amount days.

However, it may be a liability to maintain these kinds of details given that they're liable to change.

**Comment by Madhawa97** [comment]
Exactly. I think It's better to create a new table for each of these and add a reference field to the main table (link to the detail table).
